src/accelerate/components/List.tsx

```diff
@@ -110,16 +110,15 @@ export default function List( props: Props ) {
 								timer && clearTimeout( timer );
 								timer = setTimeout( value => {
 									setSearch( value );
+									analytics.track(
+										'filter',
+										{
+											location: 'dashboard',
+											filter_type: 'search',
+											filter_value: e.target.value,
```

Comment:
Weirdly `e.target.value` might actually be different than what is intended here, this is why you can pass values to the `setTimeout` callback, use `value` instead:
```suggestion
											filter_value: value,
```